### PR TITLE
Release 1.10.3

### PR DESCRIPTION
**Pull request by filipenevola**
Check here the [milestone](https://github.com/meteor/meteor/milestone/72)

- [x] MongoDB Update - it was reverted until we fix this issue https://github.com/meteor/meteor/pull/11100#discussion_r459989574

closed in favor of [1.11](https://github.com/meteor/meteor/pull/11144)

### CODE REVIEW

**Comment by hexsprite** [comment]
Getting this error when trying to upgrade to beta3:

```
=> Errors while initializing project:

While selecting package versions:
error: Conflict: Constraint email@1.2.3 is not satisfied by email 2.0.0-beta1103.3.
Constraints on package "email":
* email@2.0.0-beta1103.3 <- top level
* email@~2.0.0-beta1103.3 <- top level
* email@1.2.3 <- accounts-password 1.6.1
* email@1.0.7 <- montiapm:agent 2.38.1

Conflict: Constraint email@1.0.7 is not satisfied by email 2.0.0-beta1103.3.
Constraints on package "email":
* email@2.0.0-beta1103.3 <- top level
* email@~2.0.0-beta1103.3 <- top level
* email@1.2.3 <- accounts-password 1.6.1
* email@1.0.7 <- montiapm:agent 2.38.1
```

Particularly interesting is the error from `accounts-password` since it's an internal package shouldn't it be compatible with email?

**Comment by StorytellerCZ** [comment]
`accounts-password` needs to be updated with the latest version constraint...

File: History.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+### Breaking changes
+
+* `email` package dependencies have been update and package version has been bumped to 2.0.0
+    There is a potential breaking change as the underlying package started to use `dns.resolve()`
+    instead of `dns.lookup()` which might be breaking on some environments.
+    See [nodemailer changelog](https://github.com/nodemailer/nodemailer/blob/master/CHANGELOG.md) for more information.
+
+### Changes
+
+* `email` package now exposes `hookSend` that runs before emails are send.
```
Comment (wreiske) [inline History.md:12]:
```suggestion
* `email` package now exposes `hookSend` that runs before emails are sent.
```

File: packages/accounts-base/accounts_server.js [hunk 2]
```diff
@@ -117,6 +117,19 @@ export class AccountsServer extends AccountsCommon {
     this._validateNewUserHooks.push(func);
   }
 
+  /**
+   * @summary Validate login from external service
+   * @locus Server
+   * @param {Function} func Called whenever login/user creation from external service is attempted. Login or user creation based on this login can be aborted by by passing a falsy value or throwing an exception.
```
Comment (wreiske) [inline packages/accounts-base/accounts_server.js:123]:
```suggestion
   * @param {Function} func Called whenever login/user creation from external service is attempted. Login or user creation based on this login can be aborted by passing a falsy value or throwing an exception.
```

File: packages/email/email.js [hunk 3]
```diff
@@ -98,15 +98,15 @@ var smtpSend = function (transport, mail) {
   transport._syncSendMail(mail);
 };
 
+var sendHooks = [];
 /**
- * Mock out email sending (eg, during a test.) This is private for now.
+ * Hook that runs before email is send.
```
Comment (wreiske) [inline packages/email/email.js:103]:
```suggestion
 * Hook that runs before email is sent.
```

**Comment by filipenevola** [comment]
Hey @hexsprite @StorytellerCZ this is fixed on beta.5

> meteor update --release 1.10.3-beta.5

**Comment by arggh** [comment]
Is there any chance of getting https://github.com/meteor/meteor/issues/10903 (support for pnpm) fixed for `1.10.3`? There's not that much content in the [1.10.3 milestone](https://github.com/meteor/meteor/milestone/72) and we still accept candidates beforce the release candidate -phase, right?

It's a bummer that you can't use pnpm with Meteor, meaning Meteor's bundler doesn't actually conform to the node module resolution rules and Meteor devs are missing out on this great package manager!

**Comment by filipenevola** [comment]
Hey, Meteor 1.10.3-beta.6 includes Node and NPM updated versions. Please test your project and provide feedback.

File: packages/accounts-base/accounts_tests.js [hunk 4]
```diff
@@ -623,3 +623,44 @@ Tinytest.add(
     Accounts._options = accountsOptions;
   }
 );
+
+Tinytest.add(
+    'accounts - verify beforeExternalLogin hook can stop user login',
```
Comment (hexsprite) [inline packages/accounts-base/accounts_tests.js:628]: seems like this is not documented or added to the change notes

Comment (filipenevola) [reply packages/accounts-base/accounts_tests.js:628]: @hexsprite this is documented here https://github.com/meteor/docs/pull/677

Comment (hexsprite) [reply packages/accounts-base/accounts_tests.js:628]: thank you. seems like all it needs a references in CHANGES then

**Comment by StorytellerCZ** [comment]
APM will have to be re-published after release. Testing things out now.
```
➜  app git:(develop) ✗ meteor update --release 1.10.3-beta.6
=> Errors while initializing project:                                              
                                              
While selecting package versions:
error: Conflict: Constraint email@1.2.3 is not satisfied by email 2.0.0-beta1103.6.
Constraints on package "email":
* email@~2.0.0-beta1103.6 <- top level
* email@1.2.3 <- mdg:meteor-apm-agent 3.2.6
* email@2.0.0-beta1103.6 <- accounts-password 1.6.2-beta1103.6
```

**Comment by wreiske** [comment]
I've built a GitHub action for MongoDB builds, more info here https://github.com/meteor/meteor/pull/11100#issuecomment-664553951

**Comment by filipenevola** [comment]
As we are updating MongoDB Driver and Node.js we decide to call it [1.11](https://github.com/meteor/meteor/pull/11144)

**Pull request closed without merging**